Slash/Utility/Comments/Comments.pm

```diff
@@ -1345,44 +935,21 @@ sub displayThread {
 			$class = 'hidden';
 		} elsif ($comment->{points} < $threshold) {
```

Comment:
get rid of this elsif.  doen in template.  also any other shit to do with setting classes.